File: Codeforces/NextRound.py
import array
n, k = list(map(int, input().split()))
arr = [n]
for i in range(0, n):
    arr[i] = int(input())
temp = arr[k-1]
count=0
for i in range(0, n):
    if arr[i]>=temp and arr[i]!=0:
        count+=1
print(count)

File: Codeforces/NextRound.cpp
#include "bits/stdc++.h"
using namespace std;
int main(){
    int n, k; cin>>n>>k;
    int arr[n];
    for(int i=0; i<n; i++){
        cin>>arr[i];
    }
    int temp = arr[k-1];
    int count=0;
    for(int i=0 ;i<n; i++){
        if(arr[i]>=temp && arr[i]!=0){
            count++;
        }
    }
    cout<<count<<"\n";
    return 0;
}